orgbee/src/app/pages/events/events.component.html

```diff
@@ -1 +1,112 @@
-<p>events works!</p>
+<div class="events-page">
+  <div class="top">
+    <app-header [title]="'Events'" 
+      [showImage]="false" 
+      [height]="'shorter-height'">
+    </app-header>
+
+    <button 
+      id="createEventBtn" 
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. See the link for reference. 
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5